fix-code-0/main.cpp

```diff
@@ -3,5 +3,7 @@
 using namespace std;
 
 int main() {
-  cout<<"Hello "; cout <<"World!"; cout << endl;
+  cout << "Hello "; 
+  cout << "World!"; 
```

Comment:
Instead of writing a new line for cout << endl, can we add it to the end of each sentence?  Ex: "cout << "I like learning new things like coding!" << endl;

Reply:
Done.